Backend/app/provider-platform/dynamic-offer-driver-app/Main/src/Domain/Action/UI/Payment.hs

```diff
@@ -97,7 +98,8 @@ createOrder (driverId, merchantId, opCityId) invoiceId = do
     CQSC.findSubscriptionConfigsByMerchantOpCityIdAndServiceName opCityId serviceName
       >>= fromMaybeM (NoSubscriptionConfigForService opCityId.getId $ show serviceName)
   let paymentServiceName = subscriptionConfig.paymentServiceName
-  (createOrderResp, _) <- SPayment.createOrder (driverId, merchantId, opCityId) paymentServiceName (catMaybes driverFees, []) Nothing INV.MANUAL_INVOICE (getIdAndShortId <$> listToMaybe invoices) Nothing
+  vendorFees <- mapM QVF.findByDriverFeeId (map Domain.Types.DriverFee.id (catMaybes driverFees))
```

Comment:
bro, can we have a flag in subsconfig for this query :-
`vendorFees <- mapM QVF.findByDriverFeeId (map Domain.Types.DriverFee.id (catMaybes driverFees))`
like :- 
`if isEnabledVendorSplit then mapM QVF.findByDriverFeeId (map Domain.Types.DriverFee.id (catMaybes driverFees)) else []`